pythonsd/templates/pythonsd/base.html

```diff
@@ -23,9 +23,14 @@
 
   <body>
     <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-      <a class="navbar-brand" href="{% url 'index' %}">
-        <img src="{% static 'img/sandiegopython-logo.svg' %}" alt="San Diego Python Home">
+    {% url 'index' as index_url %}
+    {% if request.path == index_url %}
+		<img src="{% static 'img/sandiegopython-logo.svg' %}" id="navbar-logo" class="navbar-brand" alt="San Diego Python Home">
+    {% else %}
+      <a class="navbar-brand" href="{{ index_url }}">
+		<img src="{% static 'img/sandiegopython-logo.svg' %}" id="navbar-logo" alt="San Diego Python Home">
       </a>
+    {% endif %}
```

Comment:
I prefer the following to duplicating the image tag:

```suggestion
      {% url 'index' as index_url %}
      {% if request.path != index_url %}<a class="navbar-brand" href="{{ index_url }}">{% endif %}
		  <img src="{% static 'img/sandiegopython-logo.svg' %}" id="navbar-logo" class="navbar-brand" alt="San Diego Python Home">
      {% if request.path != index_url %}</a>{% endif %}
```

Reply:
If the CSS screenshot is from this code, that is where the 10px is coming from. This would have added the padding for the navbar-brand class twice, <a> and <img>.